routes/login/login.controller.js

```diff
@@ -1,27 +1,63 @@
-module.exports = function(app) {
+module.exports = function (app) {
   // redirect from "/login" → "/login/accessCode"
   app.get('/login', (req, res) => res.redirect('/login/code'))
   app.get('/login/code', (req, res) =>
     res.render('login/code', { title: 'Enter access code', data: req.session || {} }),
   )
-  app.post('/login/code', postCode)
+  app.post('/login/code', processCode)
   app.get('/login/success', (req, res) =>
     res.render('login/success', { title: 'Your access code', data: req.session || {} }),
   )
+
+  //SIN
+  app.get('/login/sin', (req, res) =>
+    res.render('login/sin', { title: 'Enter your SIN', data: req.session || {} }),
+  )
+  app.post('/login/sin', processSIN)
+
+  app.get('/login/sin', (req, res) =>
+    res.render('login/sin', { title: 'Enter your SIN', data: req.session || {} }),
```

Comment:
Could we expand the acronym in the title? Our new mockups have this, and in general, we should be referring to the whole thing the first time we mention it.

Reply:
I concur
![concur-my-friend-i-concur-memes-com-16169910](https://user-images.githubusercontent.com/289193/60729398-99caab00-9f10-11e9-9f58-bb74a39a62df.png)